lib/ec2imgutils/ec2utils.py

```diff
@@ -166,11 +199,23 @@ def get_from_config(account, config, region, entry, cmd_line_arg):
             raise EC2AccountException(msg % cmd_line_arg)
 
         account_name = generate_config_account_name(account)
-        try:
-            value = config.get(account_name, entry)
-        except Exception:
-            msg = 'Unable to get %s value from account section %s'
-            raise EC2AccountException(msg % (entry, account))
+        if entry in ['secret_access_key', 'access_key_id']:
+            try:
+                value = config.get(account_name, entry)
+            except Exception:
+                try:
+                    value = get_account_info_from_aws(account, entry)
+                except Exception:
+                    msg = 'Unable to get %s value from the aws credentials '
+                    msg += 'for region %s.'
```

Comment:
Now that we are looking in multiple places we want the user to know where we looked, i.e something like

"Unable to determine the secret_access_key from ~/.ec2utils.conf, ~/.aws/config, ~/.aws/credentials for account/profile XYZ"

The region information is not relevant here.

Reply:
Thansk for spotting this. I agree we should list all files searched. I will change the msg.